docs/pressable.md

```diff
@@ -15,14 +15,17 @@ Pressable is a Core Component wrapper that can detect various stages of press in
 
 On an element wrapped by `Pressable`:
 
-1. [`onPressIn`](#onpressin) is called when a press is activated—before `onPress` is called.
-2. [`onPress`](#onpress) is called when a single press gesture is triggered, 130 milliseconds from `onPressIn`.
-3. [`onLongPress`](#onlongpress) is called only if the press gesture is activated beyond 500ms from `onPressIn` or the time set with [`delayLongPress`](#delaylongpress).
-4. [`onPressOut`](#onpressout) is called when the press gesture is deactivated.
+- [`onPressIn`](#onpressin) is called when a press is activated.
+- [`onPressOut`](#onpressout) is called when the press gesture is deactivated.
 
-  <img src="/docs/assets/d_pressable_pressing.svg" width="1000" alt="Diagram of the onPress events in sequence.">
+After pressing [`onPressIn`], one of two things will happen:
 
-Fingers are not the most precise instruments, and it's not uncommon for users to "fat finger" an interface—to activate the wrong thing or miss the activation area. To help, `Pressable` has an optional `HitRect` you can use to define how far a touch can register away from the the wrapped element. Presses can start anywhere within a `HitRect`.
+1. The person will remove their finger, triggering [`onPressOut`] followed by [`onPress`](#onpress).

```

Comment:
Should onPressOut here be a link as well? There are a couple of other ones missing links as well

Reply:
You only have to link on the first use of a term.